controller/blogy.js

```diff
@@ -0,0 +1,62 @@
+const Blog = require("../model/post");
```

Comment:
Your names are still conflicting, `Blog --- /model/post`, you need to pick one

Reply:
I changed it to Post